homework/anastasiadom/images.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import uuid
+
+urlList = ['http://www.mmf.lnu.edu.ua/media/k2/items/cache/537eafaab0a4c0a227d88cc02e4492cf_XL.jpg',
+			'https://habr.com/post/210238/' ,
+			'http://docs.python-requests.org/en/master/_static/requests-sidebar.png',
+			'http://www.mmf.lnu.edu.ua/ar/1743',
+			'http://www.mmf.lnu.edu.ua/images/stories/articles/181015m28.jpg',
+			'https://images.unsplash.com/photo-1535498730771-e735b998cd64?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80']
+
+typeList = ['image/jpeg','image/png']
+
+class ImageDownloader:
+
+	DEFAULT_TYPES = ['image/jpeg','image/png','image/jpg', 'image/gif', 'image/psd']
+	DEFAULT_MAX_FILE_SIZE = 5242880
+
+	def __init__(self, urlList = [], typeList = DEFAULT_TYPES, size = DEFAULT_MAX_FILE_SIZE):
```

Comment:
DEFAULT_TYPES, DEFAULT_MAX_FILE_SIZE that you already have as class variables, you do not need to set its into __init__. Use self. DEFAULT_TYPES or css. DEFAULT_TYPES to get it

Reply:
1. DEFAULT_TYPES removed from parameters
2. DEFAULT_MAX_FILE_SIZE is added as optinal parameter (you can set any file size as second argument but by default it equal DEFAULT_MAX_FILE_SIZE )